packages/catalog-api/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@webcomponents/catalog-api",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Schema and interfaces for the @webcomponents/catalog-server",
+  "author": "Google LLC",
+  "homepage": "",
+  "license": "APACHE-2.0",
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "build": "wireit",
+    "build:ts": "wireit",
+    "build:graphql": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "command": "cp -r out/src/* . ; cp -r out/gen/* .",
+      "dependencies": [
+        "build:graphql",
+        "build:ts"
+      ],
+      "files": [
+        "out/"
+      ],
+      "output": [
+        "lib",
+        "index.{js,js.map,d.ts,d.ts.map}"
+      ],
+      "clean": "if-file-deleted"
+    },
+    "build:ts": {
+      "command": "../../node_modules/.bin/tsc",
```

Comment:
why do we keep doing this instead of just `tsc`? npm will resolve it fine, right?

Reply:
Oh, if that works, then yeah, I'll change it.